### ISSUE TITLE
Tests break after upgrading from 0.1.15 to 0.2.0

### ISSUE THREAD

**Issue by janhesters** [onflow/flow-js-testing#65]
### Problem

After upgrading to `0.2.0` my tests break. (BTW some tests in your pipeline should've failed and blocked the merge, too, e.g. `"executeScript - shall work properly for empty array as argument"` in `/dev-test/interaction.test.js`.)

### Steps to Reproduce

`flow.json`:

```json
{
    "accounts": {
        "emulator-account": {
            "address": "f8d6e0586b0a20c7",
            "key": "$PRIVATE_KEY"
        }
    },
    "contracts": {},
    "deployments": {},
    "emulators": {
        "default": {
            "port": 3569,
            "serviceAccount": "emulator-account"
        }
    },
    "networks": {
        "emulator": "127.0.0.1:3569",
        "mainnet": "access.mainnet.nodes.onflow.org:9000",
        "testnet": "access.devnet.nodes.onflow.org:9000"
    }
}
```

Contract (in `cadence/contracts/HelloWorld.cdc`):

```cdc
pub contract HelloWorld {
  pub let greeting: String

  init(greeting: String){
    self.greeting = greeting
  }
}
```

Test:

```ts
/**
 * @jest-environment node
 */

import {
  deployContractByName,
  emulator,
  executeScript,
  init,
} from 'flow-js-testing';
import path from 'path';

describe('flow sanity checks', () => {
  beforeEach(async () => {
    // eslint-disable-next-line unicorn/prefer-module
    const basePath = path.resolve(__dirname, '../../cadence');
    // You can specify different port to parallelize execution of describe blocks
    const port = 8080;
    await init(basePath, { port });

    // Setting logging flag to true will pipe emulator output to console
    const logging = false;
    return emulator.start(port, logging);
  });

  afterEach(async () => {
    // Stop emulator, so it could be restarted
    return emulator.stop();
  });

  describe('hello world contract', () => {
    it('given a greeting: sets the greeting', async () => {
      const greeting = 'Hello from Emulator';
      await deployContractByName({ name: 'HelloWorld', args: [greeting] });

      const actual = await executeScript({
        code: `
        import HelloWorld from 0x1
        
        pub fun main(): String{
          return HelloWorld.greeting
        } 
      `,
      });
      const expected = greeting;

      expect(actual).toEqual(expected);
    });
  });
});
```

Before the upgrade the test passes. After the upgrade it fails with:

```
  ● flow sanity checks › hello world contract › given a greeting: sets the greeting

    expect(received).toEqual(expected) // deep equality

    Expected: "Hello from Emulator"
    Received: [null, [Error: [Error Code: 1101] cadence runtime error Execution failed:
    error: unexpected token: ','
     --> 5cb96ebdb1396b4fbf617912c5131a62fa91558cffa4eb805875071303de0a85:2:49
      |
    2 |         import HelloWorld from 0xf8d6e0586b0a20c7,
      |                                                  ^
    ]]

      56 |       expect(actual).toEqual(expected);
      57 |     });
    > 58 |   });
         |                ^
      59 | });
      60 |

      at Object.<anonymous> (src/features/flow/flow-sanity-check.test.ts:58:28)
```

Obviously, changing `actual` to `[actual]` because of the new array return doesn't fix the issue, because there is an error in the second value of the array and the first is `null`.

### Acceptance Criteria

Tests don't break, or more specifically, there are no random `,` inserted in parsed code.

### Context

<what are you currently working on that this is blocking?>

I'm exploring flow and play around with the tooling. So far I love it, but I had to downgrade to 0.1.15 to keep working.

**Comment by MaxStalker** [comment]
Thank you for your kind words, spotting and flagging this issue! :bow: 
Fix is on the way - https://github.com/onflow/flow-js-testing/pull/66

I will add additional tests and refactor dev-tests to prevent merging as well in near future.

**Comment by janhesters** [comment]
@MaxStalker Wow, you're fast. Thank you!

Random (off-topic) question (I hope you don't mind 😇 ), is there a possibility to test if values get correctly logged out?

For example, if I have a transaction `cadence/transactions/log-signers.cdc` like this:

```cdc
transaction(message: String){
  prepare(first: AuthAccount, second: AuthAccount){
      log(message)
      log(first.address)
      log(second.address)
  }
}
```

and run it in a test like this:

```ts
const Alice = await getAccountAddress('Alice');
const Bob = await getAccountAddress('Bob');
const message = 'Hello from Emulator';
const signers = [Alice, Bob];

await sendTransaction({
  name: 'log-signers',
  args: [message],
  signers,
});
```

Is there a way to capture the logs (so you can do assertions on it within the test)?

**Comment by MaxStalker** [comment]
=^__^=

Great idea!
I want to do more with logs as well, but unfortunately it's not possible at this very moment.
Can you fill `feature request` here - I might get back to it soon :)

**Comment by janhesters** [comment]
Will do! Thank you very much 🙏

**Comment by MaxStalker** [comment]
@janhesters Fix deployed with version `0.2.1` now - please let me know if you experience any difficulties with it 🙇

**Comment by janhesters** [comment]
@MaxStalker works perfectly!
